xbmc/interfaces/legacy/ListItem.cpp

```diff
@@ -482,6 +482,8 @@ namespace XBMCAddon
             videotag.SetTrailer(value);
           else if (key == "path")
             videotag.SetPath(value);
+          else if (key == "filenameandpath")
```

Comment:
I don't know all about our inner workings, but this change doesn't seem related, is it?